onnx/backend/test/case/node/reducesum.py

```diff
@@ -194,3 +194,27 @@ def export_empty_axes_input_noop() -> None:
             outputs=[reduced],
             name="test_reduce_sum_negative_axes_keepdims_random",
         )
+
+    @staticmethod
+    def export_empty_set() -> None:
```

Comment:
Maybe that would be worth adding example like `ReduceSum([[], []], axis=0)` and `ReduceSum([[], []], axis=1)`.

Reply:
Done